virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -0,0 +1,175 @@
+
+### Tasks (VirtualBox)
+
+- Download the ISO image from the [Ubuntu homepage](https://ubuntu.com/download/server. Please choose the "20.04.3 LTS" image which is called "ubuntu-20.04.3-live-server-amd64.iso"
+- While the image is being downloaded, you can start preparing the VM already. Create a new VM in VirtualBox called "Ubuntu Server" and assign 2048 MB of memory to it. If that is not possible on your machine, choose a larger or smaller amount of memory. Make sure the your virtual hard drive is large enough. 6 GB should be enough for this exercise. If you choose "dynamic allocation" the actual image size should not grow beyond 4 GB during this exercise. So it you are very limited on disk space you can also try a smaller disk size than 6 GB.
+- Mount the `ubuntu-20.04.3-live-server-amd64.iso` to your virtual machine's cdrom drive (Storage -> Controller: IDE), make sure the graphics card has at least 32 MB video memory and start the VM.
+- The VM will boot from the Ubuntu image and will welcome you with the a configuration program. Choose your preferred language and then choose "Start Ubuntu Server". This will boot Ubuntu and bring you to the installation routine. The boot process might take a short while since Ubuntu will generate some files and run some checks.
+- When the boot process has finished you will be presented with Ubuntu's installation program. Go through it carefully.
+  - Choose your preferred language and keyboard layout. Note that you cannot use your mouse in the menus instead you have to use your keyboard.
+  - For most settings you can accept the default values Ubuntu suggests (mirror, empty proxy, using entire disk with LVM group, storage configuration etc.)
+  - Confirm formatting of the virtual hard drive ("Confirm destructive action") if asked so.
+- When setting up your user, please use your GitLab username, e.g. `jaustar`, as username and choose the username + VM, e.g. `jaustarvm`, as the server's name. The password you can choose freely. Note that the password cannot be empty. If you do not feel creative, use `vm` as password.
+- When you are asked about the SSH Setup, you can decide whether you want to activate the `Install OpenSSH Server` option (using spacebar) or not. If you want to test out the SSH connection to the VM (optional task), you can activate it now. If you do not activate it here, you can install the OpenSSH server later as well.
+- In the next window you skip all suggested "Featured Server Snaps" to keep the size of the image minimal. It suggests common software used on the Ubuntu Server edition, such as Docker, for example. However, we do not need it here. Confirming your choice will start the installation procedure.
+- The installation procedure might take a while since it will also install security updates. You can already start reading on the subsequent sections.
```

Comment:
Some visual break after this item would be good.

Reply:
I will rewrite the exercise while keeping this in mind.